src/hyperion/external_interaction/callbacks/rotation/zocalo_callback.py

```diff
@@ -26,7 +26,7 @@ def __init__(
 
     def start(self, doc: dict):
         LOGGER.info("Zocalo handler received start document.")
-        if doc.get("subplan_name") == "rotation_scan_main":
+        if self.run_uid is None:
```

Comment:
Must: This is all about making sure the `isypb` deposition happens before `zocalo` triggers, right? I think we need a very explicit test that confirms this, otherwise we'll end up accidentally hitting it again